Plugin.cs

```diff
@@ -77,11 +78,16 @@ public void NeedCommand(string command, string args)
                         RollItem(RollOption.Need, index);
                         ++num1;
                     }
-                    else
+                    else if (!LootItems[index].Rolled)
                     {
                         RollItem(RollOption.Greed, index);
                         ++num2;
                     }
+                    else
+                    {
+                        RollItem(RollOption.Pass, index);
+                        ++num3;
+                    }
                 }
             }
             if (!config.EnableChatLogMessage)

```

Comment:
Hey, it is not related to your commit itself, but is this line being stored when we check/uncheck the checkbox in the config? I don't think it was. Could you look at it? Thanks for the PR, you rockl!

Reply:
I did look into the config not saving the checkbox selection and it seems like the code doesn't save any changes to it. I'll make a pr when I'm able to fix it.